0003-automatic-telemetry-libraries.md

```diff
@@ -0,0 +1,66 @@
+# Automatic language/framework instrumentation in OpenTelemetry
+
+_Cross-language requirements for automatically extracting portable telemetry data with minimal ("greybox") source code modification._
```

Comment:
I'm very on board with minimal, "greybox" implementation. However, I think using the word "automatic" is confusing: it implies zero-touch, rather than low-touch.